web/src/main/java/org/wso2/testgrid/web/plugins/AWSArtifactReader.java

```diff
@@ -34,37 +37,29 @@
  */
 public class AWSArtifactReader implements ArtifactReadable {
 
-    private static final String AWS_ACCESS_KEY = "AWS_ACCESS_KEY_ID";
-    private static final String AWS_SECRET_KEY = "AWS_SECRET_ACCESS_KEY";
     private final AmazonS3 amazonS3;
     private final String bucketName;
 
     /**
      * Creates an instance of {@link AWSArtifactReader} for the given region and bucket name.
      * <p>
-     * Please do note that the {@value AWS_ACCESS_KEY} and {@value AWS_SECRET_KEY} environment values should be set
+     * Please do note that accessKey and secretKey values should be set in a properties file
      * in order to authenticate to the AWS.
      *
      * @param region region where the S3 bucket is located
      * @param bucket name of the bucket
-     * @throws ArtifactReaderException thrown if the given parameters are null or empty or if
-     *                                 the {@value AWS_ACCESS_KEY} and {@value AWS_SECRET_KEY} environment variable
-     *                                 values are not set
+     * @throws ArtifactReaderException thrown if the given parameters are null or empty
      */
     public AWSArtifactReader(String region, String bucket) throws ArtifactReaderException {
-        String awsIdentity = System.getenv(AWS_ACCESS_KEY);
```

Comment:
We could have use the existing configuration context to get values from property file as well. :)
`String awsIdentity = ConfigurationContext.getProperty(AWS_ACCESS_KEY_ID);`

Reply:
PropertiesFileCredentialsProvider type of AWSCredentialsProvider takes the property file as an argument. Reading the file and extracting the property is handled by the PropertiesFileCredentialsProvider. It is not needed to extract the property value from the file. IMO, ConfigurationContext.getProperty is not necessary here. WDYT?

Reply:
Yes, true :+1: